polygeist/tools/cgeist/Test/Verification/sycl/kernels_funcs.cpp

```diff
@@ -17,12 +17,10 @@
 // CHECK-MLIR-SAME: attributes {llvm.cconv = #llvm.cconv<spir_funccc>, llvm.linkage = #llvm.linkage<linkonce_odr>, {{.*}}
 
 // COM: StoreWrapper constructor:
-// CHECK-LLVM-DAG:      define linkonce_odr spir_func void @_ZN12StoreWrapperIiLi1ELN4sycl3_V16access4modeE1026EEclEv({{.*}}) #[[FUNCATTRS2:[0-9]+]]
```

Comment:
I'd like to make the optimization level used explicit. Can you pls add -O0 to the "RUN" lines? 
Eventually we will want to also run the test cases with -O2, -O3, -Ofast".... so adding the opt level explicitly will allow that.

Same comment for all SYCL tests we have. All of them should have the opt level specified explicitly. 
We could add a new separate lit test to check what the expected default opt. level is (-O0).